bot/seasons/valentines/movie_generator.py

```diff
@@ -0,0 +1,62 @@
+import logging
+import random
+from os import environ
+from urllib import parse
+
+import aiohttp
+import discord
+from discord.ext import commands
+
+log = logging.getLogger(__name__)
+
+
+class MovieGenerator:
+    """
+    a cog that gives random romance movie suggestion to a user
+
+    """
+
+    def __init__(self, bot):
+        self.bot = bot
+
+    @commands.command()
+    async def romancemovie(self, ctx):
```

Comment:
I don't really like this function name, maybe name the function `romance_movie` and have an alias in the `@commands.command()` line so it is like `@commands.command(aliases=["romancemovie"])`
```suggestion
    async def romance_movie(self, ctx):
```